Makefile

```diff
@@ -3,7 +3,10 @@
 #
 
 ifndef ZEPHYR_BASE
-$(error The ZEPHYR_BASE environment variable must be set)
+WEST := $(shell sh -c "command -v west")
+ifeq ($(strip $(WEST)),)
+$(error The ZEPHYR_BASE environment variable is not set, and west was not found)
```

Comment:
This check duplicates the one in `doc/CMakeLists.txt`, let's get rid of it.

Reply:
Done, thanks for review!